flockoff/utils/git.py

```
@@ -1,86 +1,85 @@
+import os
 import subprocess
+import sys
+import configparser
 import bittensor as bt
+from flockoff import constants
```

Ruff (0.11.9)

`flockoff.constants` imported but unused

Remove unused import: `flockoff.constants`

(F401)